radio/src/mixer.cpp

```diff
@@ -384,6 +384,13 @@ getvalue_t getValue(mixsrc_t i, bool* valid)
 
   else if (i <= MIXSRC_LAST_TRIM) {
     i -= MIXSRC_FIRST_TRIM;
+    if (getRawTrimValue(mixerCurrentFlightMode, i).mode == TRIM_MODE_3POS) {
+      // Trim set as 3POS toggle switch in FM
+      uint8_t tidx = i * 2;
+      if (trimDown(tidx)) return -1024;
+      else if (trimDown(tidx + 1)) return 1024;
```

Comment:
Just nit-picking here, but didn't we have some constant for such min/max values?

Reply:
Thanks, fixed now.